CONFIGURE.sh

```diff
@@ -2,12 +2,29 @@
 
 ########################################################################################################################
 ########################################################################################################################
-# this script must be run by root or sudo 
+# this script must be run by root or sudo
 if [[ "$UID" -ne "0" ]] ; then
     echo "ERROR: This script must be run by root or sudo"
     exit
 fi
 
+echo -e "\nWelcome to the Homework Submission Server (HSS) Configuration\n"
+
+echo "What is the database host?"
+read DATABASE_HOST
+
+echo "What is the database user?"
+read DATABASE_USER
+
+echo "What is the database password for the database user $DATABASE_USER?"
+read DATABASE_PASSWORD
+
+echo "What is the url for the Grading Server?"
+read TAGRADING_URL
+
+echo "What is the SVN path to be used?"
```

Comment:
The SVN path is hacky right now, only works for one class.  Can we delay putting the prompt in until that's actually properly generalized?